jobs/JGLOBAL_WAVE_PREP

```diff
@@ -79,13 +79,24 @@ else
       $NLN $DMPDIR/${WAVECUR_DID}.${RPDY} $ROTDIR/${WAVECUR_DID}.${RPDY}
     fi
     export COMIN_WAV_RTOFS=${COMIN_WAV_RTOFS:-$ROTDIR}
-  fi 
+  fi
```

Comment:
Please see /scratch2/NCEPDEV/climate/Jessica.Meixner/p7update/mergedev/jobs/JGLOBAL_WAVE_PREP and update this file. 

The new section with BRPDY should also be behind the if [ $WW3CURINP = "YES" ] statement and the "COMIN_WAV_ICE" line is repeated.  The COMIN_WAV_RTOFS also needs to be in the if statement block.